### ISSUE TITLE
Code cleanup: standardize JavaScript locations and naming scheme

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2292]
I noticed that we have no standardized way of naming our JavaScript files: sometimes we capitalize the first letter (other times not), sometimes we use dots to separate words other times we use CamelCase, etc. We should standardize this, fix it, and update our style guide.

Similarly, we should fix *where* we put our JavaScript files. Currently, some are in sub-dirs in `javascripts` like `javascripts\Progress\Progress.js` while other times they are in the root `javascripts folder.

**Comment by jonfroehlich** [comment]
Still work to do here?

**Comment by misaugstad** [comment]
Yes. I did a large portion of this already by moving all the libraries to the `public/javascripts/lib` directory (and getting rid of the `public/javascripts/SVLabel/lib` directory) and by removing all the old libraries and code we aren't using anymore. Things feel a lot more clean and organized now.

At the very least, the "Progress" directory and scripts needs to be renamed to something like UserProfile though. And the directory structure is different for different sub-directories of `public/javascripts`. And there are still a bunch of scripts that aren't in a subdirectory of `public/javascripts`; I'm not totally sure if we should create separate directories for some of those.

**Comment by misaugstad** [comment]
Closing via #4468
